src/generator/AutoRest.Ruby.Azure/AzureRubyCodeGenerator.cs

```diff
@@ -93,14 +93,25 @@ private void AddRubyPageableMethod(ServiceClient serviceClient)
                     PageableExtension pageableExtension = JsonConvert.DeserializeObject<PageableExtension>(method.Extensions[AzureExtensions.PageableExtension].ToString());
                     if (pageableExtension != null && !method.Extensions.ContainsKey("nextLinkMethod") && !string.IsNullOrWhiteSpace(pageableExtension.NextLinkName))
```

Comment:
>!method.Extensions.ContainsKey("nextLinkMethod") [](start = 53, length = 48)

should we be fine removing `!method.Extensions.ContainsKey("nextLinkMethod")` condition?